package.json

```diff
@@ -41,5 +41,8 @@
     "prettier": "prettier",
     "tsc": "tsc",
     "multiapi": "ts-node ./scripts/multiapi.ts"
+  },
+  "dependencies": {
+    "swagger-model-validator": "^3.0.21"
```

Comment:
why do you add this change in the top level package.json file ?

Reply:
this is a mistake. I will revert.